src/test/java/StorageTest.java

```diff
@@ -0,0 +1,28 @@
+import org.junit.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
```

Comment:
Missing header comment for StorageTest class, AKA javadocs. (Under Comments in coding standard)